manifests/install.pp

```diff
@@ -1,7 +1,9 @@
 # == Class keepalived::install
 #
 class keepalived::install {
-  package { $keepalived::pkg_list:
-    ensure => $keepalived::pkg_ensure,
+  if $keepalived::package_manage == true {
```

Comment:
you could also use:
```suggestion
  if $keepalived::package_manage {
```

it's just shorter. But you don't have to update. We currently don't have best practice for either of those styles.

Reply:
Done.